pkg/cable/libreswan/libreswan.go

```diff
@@ -682,5 +682,5 @@ func (i *libreswan) waitForControlSocket() error {
 func (i *libreswan) Cleanup() error {
 	logger.Info("Uninstalling the libreswan cable driver")
 
-	return netlink.DeleteXfrmRules() //nolint:wrapcheck  // No need to wrap this error
+	return netlink.DeleteXfrmRules(k8snet.IPFamilyUnknown) //nolint:wrapcheck  // No need to wrap this error
```

Comment:
Overall code looks good me , but It's a bit strange that FAMILY_UNKNOWN is translated to ALL

Reply:
There's really no other choice. It can make sense depending on how you look at it.